cmd/contour/bootstrap.go

```diff
@@ -21,7 +21,9 @@ import (
 // registerBootstrap registers the bootstrap subcommand and flags
 // with the Application provided.
 func registerBootstrap(app *kingpin.Application) (*kingpin.CmdClause, *envoy.BootstrapConfig) {
-	var config envoy.BootstrapConfig
+	config := envoy.BootstrapConfig{
+		CheckFilePaths: true,
```

Comment:
This is always true right? Why have the option?

Reply:
So that testing doesn't require the actual files. I could flip the default around though?

Reply:
I always get confused about negative flags since my mind does not easily wrap around something like `dont = false`, especially when it might later get used in ways like `if dont == false { how_did_we_end_up_here? }` :-)

Reply:
Renamed to `SkipFilePathCheck`